cpp/cmake_modules/Emscripten/Platform/EmscriptenOverrides.cmake

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Force some variables for emscripten
+# to disable things that won't work there
+
+# make us be on the platforms list for cmake
+get_filename_component(PLATFORM_FOLDER_PARENT ${CMAKE_CURRENT_LIST_DIR} DIRECTORY)
+list(APPEND CMAKE_MODULE_PATH "${PLATFORM_FOLDER_PARENT}")
+
+include($ENV{EMSDK}/upstream/emscripten/cmake/Modules/Platform/Emscripten.cmake)
```

Comment:
`Emscripten.cmake` exists at `/usr/share/emscripten/cmake/Modules/Platform/Emscripten.cmake` on Debian. ( https://packages.debian.org/bookworm/all/emscripten/filelist )
So this doesn't work on Debian.

It seems that Emscripten provides `emcmake`. Can we use it like `emcmake cmake ...` instead of specifying our toolchain file by `CMAKE_TOOLCHAIN_FILE` in `CMakePresets.json`?

Reply:
Okay, have done